staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache_test.go

```diff
@@ -516,31 +572,48 @@ func TestWaitUntilFreshAndGet(t *testing.T) {
 }
 
 func TestWaitUntilFreshAndListTimeout(t *testing.T) {
-	ctx := context.Background()
-	store := newTestWatchCache(3, &cache.Indexers{})
-	defer store.Stop()
-	fc := store.clock.(*testingclock.FakeClock)
-
-	// In background, step clock after the below call starts the timer.
-	go func() {
-		for !fc.HasWaiters() {
-			time.Sleep(time.Millisecond)
-		}
-		fc.Step(blockTimeout)
-
-		// Add an object to make sure the test would
-		// eventually fail instead of just waiting
-		// forever.
-		time.Sleep(30 * time.Second)
-		store.Add(makeTestPod("bar", 5))
-	}()
-
-	_, _, _, err := store.WaitUntilFreshAndList(ctx, 5, nil)
-	if !errors.IsTimeout(err) {
-		t.Errorf("expected timeout error but got: %v", err)
+	tcs := []struct {
+		name                    string
+		ConsistentListFromCache bool
```

Comment:
This field doesn't seem to be used now :)

Reply:
Done